demos/serverless/deploy.js

```diff
@@ -11,7 +11,7 @@ let app = `meetingV2`;
 let useEventBridge = false;
 let enableTerminationProtection = false;
 let disablePrintingLogs = false;
-let chimeSDKMeetingsEndpoint = 'https://service.chime.aws.amazon.com';
+let chimeSDKMeetingsEndpoint = 'https://meetings-chime.us-east-1.amazonaws.com';
```

Comment:
Should we generate the endpoint based on region?

Reply:
Ideally yes but these values are default value and the default region value is us-east-1 so it does not make any difference.